lib/class-wp-rest-blocks-controller.php

```diff
@@ -33,4 +33,28 @@ public function check_read_permission( $post ) {
 
 		return parent::check_read_permission( $post );
 	}
+
+	/**
+	 * Retrieves the block's schema, conforming to JSON Schema.
+	 *
+	 * @since 4.4.0
+	 *
+	 * @return array Item schema data.
+	 */
+	public function get_item_schema() {
+		$schema = parent::get_item_schema();
+
+		// To use a reusable block in the editor, one needs access to its raw markup.
```

Comment:
Multiline comments should be in long comment form:

```php
/*
 * foo
 * bar
 */
```

Reply:
👍 done.